cmd/root.go

```
@@ -138,19 +138,25 @@ func init() {
-			u.LogErrorAndExit(schema.AtmosConfiguration{}, err)
+		if command.Use != "atmos" {
+			if err := oldUsageFunc(command); err != nil {
+				u.LogErrorAndExit(schema.AtmosConfiguration{}, err)
+			}
+		} else {
+			err := tuiUtils.PrintStyledText("ATMOS")
+			if err != nil {
+				u.LogErrorAndExit(schema.AtmosConfiguration{}, err)
+			}
+			b.HelpFunc(command, strings)
+			command.Usage()
```

golangci-lint (1.62.2)

Error return value of `command.Usage` is not checked

(errcheck)